UNETR/BTCV/README.md

```diff
@@ -1,3 +1,6 @@
+[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/unetr-transformers-for-3d-medical-image/medical-image-segmentation-on-synapse-multi)](https://paperswithcode.com/sota/medical-image-segmentation-on-synapse-multi?p=unetr-transformers-for-3d-medical-image)
```

Comment:
will this badge update automatically if the state-of-the-art method changes?

Reply:
Hi @wyli , It could be left un-updated. Hence, I removed it. Please review the latest PR.